docs/root/intro/arch_overview/health_checking.rst

```diff
@@ -24,6 +24,13 @@ unhealthy, successes required before marking a host healthy, etc.):
   maintenance by setting the specified key to any value and waiting for traffic to drain. See
   :ref:`redis_key <config_cluster_manager_cluster_hc_redis_key>`.
 
+Per cluster member health checking config
+-----------------------------------------
+
+If active health checking is configured for an upstream cluster, a specific additional configuration
+for each registered member can be specified by setting the
+:ref:`health check config<envoy_api_msg_endpoint.Endpoint.HealthCheckConfig>`.
```

Comment:
maybe add "in the ... message ..." ? (Basically guide the user and link to where this would be defined as part of the load assignment?) An example would be even better. :)